src/components/FileDrop/FileDrop.jsx

```diff
@@ -63,7 +63,7 @@ const propTypes = {
   /** file type either TEXT or BINARY to determine the correct encoding */
   fileType: PropTypes.oneOf(Object.values(FILE_TYPES)),
 };
-
+/* istanbul ignore next */
```

Comment:
Instead of ignoring coverage, tests should be written

Reply:
This is ignoring the default no op functions that do not need to be tested